packages/maskbook/src/social-network-adaptor/twitter.com/injection/ToolboxHint.tsx

```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles((theme) =>
             paddingBottom: theme.spacing(2),
         },
         title: {
-            color: theme.palette.mode === 'dark' ? 'rgb(216, 216, 216)' : 'rgb(15, 20, 25)',
+            color: (props) => (props.color ? props.color : 'rgb(217, 217, 217)'),
```

Comment:
```suggestion
            color: (props) => (props.color ?? : 'rgb(217, 217, 217)'),
```

And where is the `theme.palette.mode === 'dark'` part?